src/Tizen.Nlp/Tizen.Nlp/NlpCommon.cs

```diff
@@ -0,0 +1,175 @@
+/*
+* Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using Tizen.Applications;
+using Tizen.Nlp;
+
+namespace Tizen.Nlp
+{
+
+    /// <summary>
+    /// This class contains the methods and inner class related to the NLP processing.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public class NltkClass
```

Comment:
Please move this class to NltkClass.cs file. I strongly recommend that a single .cs file has only one public class.